src/tools/ScenarioMeasurement/Startup.Tests/StartupTests.cs

```diff
@@ -43,6 +46,72 @@ public void TestProfileIteration()
             TestSession(profileSession, profileParser);
         }
 
+        [Fact]
+        public void TestProcessTimeParserLinux()
+        {
+            string ctfFile = Path.Combine(testAssetDirectory, "test-process-time_startup.trace.zip");
+            var parser = new ProcessTimeParser();
+            var pids = new List<int>() { 18627, 18674, 18721, 18768, 18813 };
+            IEnumerable<Counter> counters = parser.Parse(ctfFile, "dotnet", pids, "\"dotnet\" build");
+            int count = 0;
+            foreach (var counter in counters)
+            {
+                Assert.True(counter.Results.Count == pids.Count, $"Counter {counter.Name} is expected to have {pids.Count} results.");
+                count++;
+            }
+            Assert.True(count == 1, "Only Process Time counter should be present.");
+        }
+
+        [Fact]
+        public void TestProcessTimeParserWindows()
+        {
+            string etlFile = Path.Combine(testAssetDirectory, "test-process-time_startup.etl");
+            var parser = new ProcessTimeParser();
+            var pids = new List<int>() { 32752, 6352, 16876, 10500, 17784 };
+            IEnumerable<Counter> counters = parser.Parse(etlFile, "dotnet", pids, "\"dotnet\" build");
+            int count = 0;
+            foreach (var counter in counters)
+            {
+                Assert.True(counter.Results.Count == pids.Count, $"Counter {counter.Name} is expected to have {pids.Count} results.");
+                count++;
+            }
+            Assert.True(count==2, "Both Process Time and Time To Main counter should be present.");
+        }
+
+        [Fact]
+        public void TestTimeToMainParserLinux()
+        {
+            string ctfFile = Path.Combine(testAssetDirectory, "test-time-to-main_startup.trace.zip");
+            var parser = new TimeToMainParser();
+            var pids = new List<int>() { 24352, 24362, 24371, 24380, 24389 };
+            IEnumerable<Counter> counters = parser.Parse(ctfFile, "emptycsconsoletemplate", pids, "\"pub\\emptycsconsoletemplate.exe\"");
+            int count = 0;
+            foreach (var counter in counters)
+            {
+                Assert.True(counter.Results.Count == pids.Count, $"Counter {counter.Name} is expected to have {pids.Count} results.");
+                count++;
+            }
+            Assert.True(count == 2, "Only Time To Main counter should be present.");
```

Comment:
This message and assert don't seem to add up. Is this supposed to `count == 1`?

Reply:
Good catch! It should be 1.